src/features/offerv2/components/PinchableBox/PinchableBox.tsx

```diff
@@ -0,0 +1,30 @@
+import { ReactNativeZoomableView } from '@openspacelabs/react-native-zoomable-view'
+import React from 'react'
+// eslint-disable-next-line no-restricted-imports
```

Comment:
Peut-être justifier pourquoi on ne veut pas le fastImage qui gère les images venant de notre backend ?

Reply:
On a cette explication côté Hero :
`// we import FastImage to get the resizeMode, not to use it as a component` 🤔

Reply:
Ah oui d'accord, on utilise pas le composant, juste des constantes de la lib, je vois

Reply:
C'est pas le bon commentaire en fait : on utilise bien le composant ici FastImage ici 😬